guides/06_FAQs.md

```diff
@@ -114,6 +137,10 @@ Amazon Chime SDK supports simulcast for Chromium-based browsers. See this [techn
 
 Remote mute is the ability of one attendee in a meeting to mute another attendee. This feature is currently not supported server-side by the SDK. However, you can use [Amazon Chime SDK Data Messages](https://aws.github.io/amazon-chime-sdk-js/modules/apioverview.html#9-send-and-receive-data-messages-optional) to send a message containing the attendee or attendees that should mute, and clients can mute themselves when they receive those messages.
 
+### When there are multiple monitors plugged in, my screen sharing doesn't begin unless I actively interacts with the screen
+
+This seems to be a webRTC issue and not specific to Chime SDK
```

Comment:
- `webRTC` => `WebRTC`
- add a period
- We might want to link a related WebRTC issue or GitHub issue. Here's a good exmaple.
  > This is a known issue and happens primarily due to [this Chromium bug](https://bugs.chromium.org/p/chromium/issues/detail?id=951418). At present we do not have a workaround.

Reply:
Yeah I tried to look for one but could not find any. I also do not have a multiple monitors setup so I could not reproduce this on my end.